oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/MetricsEsDAO.java

```diff
@@ -42,9 +42,9 @@
 
     @Override public Metrics get(Model model, Metrics metrics) throws IOException {
         String modelName = TimeSeriesUtils.timeSeries(model, metrics.getTimeBucket());
```

Comment:
This looks like still reading from the physical index, I think we should read from alias name, right?